drivers/soundwire/intel.c

```diff
@@ -974,11 +974,13 @@ static int intel_hw_params(struct snd_pcm_substream *substream,
 static int intel_prepare(struct snd_pcm_substream *substream,
 			 struct snd_soc_dai *dai)
 {
+	struct snd_soc_pcm_runtime *rtd = substream->private_data;
+	struct snd_soc_dai *first_cpu_dai = rtd->cpu_dais[0];
```

Comment:
@bardliao can you rebase on top of topic/sof-dev and use asoc_rtd_to_cpu(0) ? Thanks!

Reply:
> @bardliao can you rebase on top of topic/sof-dev and use asoc_rtd_to_cpu(0) ? Thanks!

Done.